{"uid":"2830a41ff34cb44","name":"test_valid_admin_login[chrome]","fullName":"Selenium_Ecommerce.tests.test_keyword.TestKeywords#test_valid_admin_login","time":{"start":1760840928886,"stop":1760840928886,"duration":0},"description":"Verify admin can log in successfully.","descriptionHtml":"<p>Verify admin can log in successfully.</p>\n","status":"skipped","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1760840928888,"stop":1760840935549,"duration":6661},"status":"broken","statusMessage":"KeyboardInterrupt\n","statusTrace":"  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\Selenium_Ecommerce\\tests\\test_keyword.py\", line 14, in setup\n    self.driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))\n                  ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 46, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        browser_name=DesiredCapabilities.CHROME[\"browserName\"],\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n        keep_alive=keep_alive,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 60, in __init__\n    self.service.start()\n    ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 110, in start\n    if self.is_connectable():\n       ~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 127, in is_connectable\n    return utils.is_connectable(self.port)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\Capstone Project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 122, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"C:\\Program Files\\Python313\\Lib\\socket.py\", line 856, in create_connection\n    exceptions.clear()  # raise only the last error\n    ~~~~~~~~~~~~~~~~^^\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0}],"afterStages":[],"labels":[{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"setup","value":"'chrome'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[],"tags":[]},"source":"2830a41ff34cb44.json","parameterValues":["'chrome'"]}